dotCMS/src/main/java/com/dotmarketing/util/ExportStarterUtil.java

```diff
@@ -0,0 +1,300 @@
+     */
+    @CloseDBIfOpened
+    private void createXMLFiles()  {
+
+        Logger.info(this, "Starting createXMLFiles into " + outputDirectory);
+
+        final Set<Class> _tablesToDump = new HashSet<Class>();
+        try {
+
+            /* get a list of all our tables */
+            //Including Identifier.class and Language.class because it is not mapped with Hibernate anymore
+            _tablesToDump.add(Identifier.class);
+            _tablesToDump.add(Language.class);
+            _tablesToDump.add(Relationship.class);
+            _tablesToDump.addAll(HibernateUtil.getSession().getSessionFactory().getAllClassMetadata().keySet());
+
+            _tablesToDump.removeIf(c->c.equals(Inode.class));
+            _tablesToDump.removeIf(c->c.equals(Clickstream.class));
+            _tablesToDump.removeIf(c->c.equals(ClickstreamRequest.class));
+            _tablesToDump.removeIf(c->c.equals(Clickstream404.class));
+            _tablesToDump.removeIf(c->c.equals(Structure.class));
+            _tablesToDump.removeIf(c->c.equals(Field.class));
+            _tablesToDump.removeIf(c->c.equals(WorkflowHistory.class));
+            _tablesToDump.removeIf(c->c.equals(PermissionReference.class));
+            
+            _tablesToDump.removeIf(t->t.getName().startsWith("Dashboard"));
+            _tablesToDump.removeIf(t->t.getName().contains("HBM"));
+
+
+            HibernateUtil _dh = null;
+            DotConnect dc = null;
+            List _list = null;
+            File _writing = null;
+            java.text.NumberFormat formatter = new java.text.DecimalFormat("0000000000");
+
+            for (Class clazz : _tablesToDump) {
+
+
+                /*
+                 * String _shortClassName =
+                 * clazz.getName().substring(clazz.getName().lastIndexOf("."),clazz.getName().length());
+                 * xstream.alias(_shortClassName, clazz);
+                 */
+                int i= 0;
+                int step = 1000;
+                int total =0;
+                
+                /* we will only export 10,000,000 items of any given type */
+                for(i=0;i < 10000000;i=i+step){
+
+                    _dh = new HibernateUtil(clazz);
+                    _dh.setFirstResult(i);
+                    _dh.setMaxResults(step);
+
+                    //This line was previously like;
+                    //_dh.setQuery("from " + clazz.getName() + " order by 1,2");
+                    //This caused a problem when the database is Oracle because Oracle causes problems when the results are ordered
+                    //by an NCLOB field. In the case of dot_containers table, the second field, CODE, is an NCLOB field. Because of this,
+                    //ordering is done only on the first field for the tables, which is INODE
+                    if(com.dotmarketing.beans.Tree.class.equals(clazz)){
+                        _dh.setQuery("from " + clazz.getName() + " order by parent, child, relation_type");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "from " appears 6 times in this file; the first occurrence is on line 144](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5632525)